apis/sgv2-docsapi/src/test/resources/application.yaml

```diff
@@ -3,3 +3,12 @@
 stargate:
   data-store:
     ignore-bridge: true
+
+"%test":
```

Comment:
Could you add a comment briefly describing the issue (that default port changed by Quarkus otherwise and must be done this way) -- so others wondering what happened can see the reference.